docs/DeveloperGuide.md

```diff
@@ -883,6 +883,61 @@ Priorities: High (must have) - `***`, Medium (nice to have) - `**`, Low (unlikel
 | `*`      | a registered owner | know the sum of all the materials required for a fixed delivery schedule | I can plan my inventory.                                                                                                                                                                |
 | `*`      | a registered owner | have different user authorisation levels                                 | I can control who has access to what.                                                                                                                                                   |
 
+### Planned Enhancements
+
+#### User Interface
+
+* The user interface can show two lists views of both customer and delivery lists side by side to allow the user to see
+  both lists at the same time.
+
+#### Command Format
+
+* Prefixes can be shortened to allow the user to type less. For example, `--name` can be shortened to `-n`.
+* Potentially frequently used commands such as `customer list` and `delivery list` can be shortened to `cl` and `dl`
+  respectively.
+
+#### Error Messages
+
+* Error messages for dates would be more specific to allow the user to know what is wrong with the date that they
+  entered. For example, if a user enters 2023-02-30, the error message would be "Invalid date. The date entered does
+  not exist.".
+* Error messages for IDs should specify whether it is for the customer ID or for delivery ID as the current
+  implementation has some error messages that only specifies "ID" instead of "Customer ID" or "Delivery ID".
+* Error messages for numerical parameters that require a positive integer should throw errors for negative values.
```

Comment:
Since it's already throwing an error for negative values, could we specify "...throw **more specific** errors for negative values." instead?

Also, do you want to consider doing a broad sweeping statement where we say error messages will be improved in general to be more informative and instructive?